magento/magento2
# #14390 issue fixed - Added storeview scope to product status

This is a bug fix for #14390 open issue reported. The problem is product can't save for storeview scope and it only works for website scope.

### Description
Added storeview scope to product status attribute

### Fixed Issues (if relevant)
1. magento/magento2#14390: Product status changed for all store_views while saving product per store_view

### Manual testing scenarios
1.Backend->Stores->Attributes->Product-> 'status' attribute code->Advanced Attribute Properties->Scope-> must have Store View in the dropdown selected after fixed the bug.
2. Backend->Edit product->Select store view (eg: Netherlands)->Untick Use Default Value and change Enable Product as No.->Save
3. Backend->Edit product->Select All Store Views-> Enable Product value should not change there and should keep its original value.
4. Backend->Edit product->Select store view (eg: Netherlands)-> Enable Product value should keep as No as changed earlier.

### Contribution checklist
